### DISCUSSION TITLE
請問Python該如何讓輸出不換行(Q.Q)?

### DISCUSSION THREAD

**Discussion by pekoranoinu** [1101-GS4719-C/Python-Community-GS4719-C#386] [Q&A] [answered]
之前學的c和c++要換行在後面加\n才會換行，但是用python print的時候都會自動換行，請問大大該怎麼做才能讓它不要換行?

**Comment by awkward-willy** [comment, marked as answer]
Python 的 print() 可以放 sep 跟 end 參數，預設的 end 是換行
要換不同的可以更改，如： print( "Hello World", end=' ' ) 就是以空格結尾

**Reply by pekoranoinu** [reply]
謝謝大解答\(>o<)/
